packages/mc-scripts/src/utils/get-session-data.spec.js

```diff
@@ -0,0 +1,63 @@
+const { rest } = require('msw');
+const { setupServer } = require('msw/node');
+
+const getSessionData = require('./get-session-data');
+
+const mockServer = setupServer();
+afterEach(() => {
+  mockServer.resetHandlers();
+});
+beforeAll(() =>
+  mockServer.listen({
+    onUnhandledRequest: 'error',
+  })
+);
+afterAll(() => mockServer.close());
+
+const mcApiUrl = 'https://mc-api.europe-west1.gcp.commercetools.com';
+
+describe('When login details are correct', () => {
+  beforeEach(() => {
+    mockServer.use(
+      rest.post(`${mcApiUrl}/tokens`, (req, res, ctx) => {
+        return res(ctx.status(200), ctx.cookie('mcAccessToken', 'hello-world'));
```

Comment:
Does this work? According to the docs, it does not seem like accessing `set-cookie` header works like this 🤔

https://mswjs.io/docs/recipes/cookies#mock-response-cookies